db/db.go

```diff
@@ -0,0 +1,35 @@
+package db
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var DB *gorm.DB
+
+func init() {
+	name := os.Getenv("MYSQL_DATABASE")
+	host := os.Getenv("DB_HOST")
+	user := os.Getenv("DB_USER")
+	pass := os.Getenv("DB_PASS")
+
+	// Formats database connection string:
+	// username:password@protocol(address)/dbname?param=value
+	dsn := fmt.Sprintf("%s:%s@tcp(%s)/%s", user, pass, host, name)
+	setupDB(dsn)
```

Comment:
You can `connect` database only using the` init func `:) i think that `setupDB` can be deleted :)